### DISCUSSION TITLE
how to change pdf title?

### DISCUSSION THREAD

**Discussion by VitaliKhileuski** [gotenberg/gotenberg#598] [Q&A]
i want to change pdf title, when i try to print my pdf document, in a tab name i have random uuid value.

**Comment by gulien** [comment]
https://gotenberg.dev/docs/modules/api#output-filename-multipart

**Reply by gulien** [reply]
If you're talking about the metadata, it's not possible via Gotenberg right now.

**Reply by VitaliKhileuski** [reply]
is it related to security? Is it worth hoping that at least the title can be changed in the future? Really need it.

**Reply by gulien** [reply]
PR welcome
